crates/sui-bridge/src/main.rs

```diff
@@ -48,6 +49,11 @@ async fn main() -> anyhow::Result<()> {
     let metadata =
         BridgeNodePublicMetadata::new(VERSION.into(), config.metrics_key_pair.public().clone());
 
+    start_metrics_push_task(
+        &config.metrics,
+        config.metrics_key_pair.copy(),
```

Comment:
do we need `copy` here?

Reply:
I believe so, `config` is also passed to `run_bridge_node(config,` on the lines below this, so without the copy I get a `value used here after partial move` compile error